ui/components/MesheryAdapterPlayComponent.js

```diff
@@ -406,8 +386,6 @@ class MesheryAdapterPlayComponent extends React.Component {
                     self.setState({ selectionError: false, cmEditorValAddError: false });
                   }
                 }}
-                onChange={(editor, data, value) => {
```

Comment:
Can you explain this a bit more, please?
I think the CodeMirror component is imported directly from node_modules. How was it's inbuild onChange handler commented?

Reply:
An empty onChange handler? Can you let me know more what function does this empty handler is supposed to perform?

Reply:
Not entirely sure, it might be useful for overriding the existing onChange function of the component. However, it is more likely it does nothing, but we need to make sure.

Reply:
This function is not called anywhere, whoever wrote this block of code copied the skeleton of the codeMirror component but later forgot to search for empty blocks & remove them. This is not the only place, if you will see #910 you will get to know how un-organised our client-side code is. This PR merely solved 59 errors that too were all because of copying of constructor props, function skeleton, setState function from other components regardless of how all the variables & parameters getting defined are even used or not.

Reply:
Can you make sure by manually testing the CodeMirror component for now? Does it works as expected?

Reply:
Tested both Codemirror & custom config application 
<img width="944" alt="Screenshot 2020-05-01 at 4 47 16 AM" src="https://user-images.githubusercontent.com/44091822/80768011-e4d04180-8b66-11ea-9d5a-372d75db961b.png">

Reply:
That's great! Thanks a lot @kushthedude